.gitignore

```diff
@@ -11,3 +11,6 @@ perf.data.old
 flamegraph.svg
 
 /rustc-ice-*.txt
+
+# hydro deploy
+.hydro/
```

Comment:
So the reason I didn't have this before is if there is a leftover folder that screams to you that you probably had a crashed run that needs resources to be cleaned up.

Reply:
I had a lot of crashed runs